packages/inferno-shared/index_out.js

```diff
@@ -0,0 +1,139 @@
+(() => {
```

Comment:
where did this file come from? 🤔

Reply:
My bad, it sneaked in when I was trying to solve the slow builds. It was generated by esbuild and should have been deleted.